src/bindings/python/flux/Makefile.am

```diff
@@ -7,7 +7,9 @@ fluxpy_PYTHON=\
 	      constants.py\
 	      jsc.py\
 	      kz.py\
-	      sec.py
+	      sec.py \
+	      job.py \
+		  mrpc.py
```

Comment:
Odd spacing here, tab/space wonkiness?

Reply:
Nice catch.  Oddly it looks fine in my browser.  I wonder how you even noticed a difference. 🤷‍♂️   It appears that all the previous lines have a single tab followed by a bunch of spaces. The last line is two tabs with fewer spaces.  I'll make that consistent. (Somehow we've ended up in the worst of both worlds with tabs vs. spaces in these automake files).